ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/capi-enabled/provision/cucushift-installer-rehearse-aws-ipi-capi-enabled-provision-chain.yaml

```diff
@@ -0,0 +1,8 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-capi-enabled-provision 
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install
+  - ref: capi-conf-apply-feature-gate        
+  documentation: |-
+    Create an IPI cluster with featuregate enabled (capi) on AWS for QE e2e tests v-4.11
```

Comment:
we’d better point out that this feature started from 4.11, we may need enable capi by featuregate for a long time, but from 4.11 begin to support this feature.

Reply:
@sunzhaohua2  PTAL , made the suggested changes .